setup.py

```diff
@@ -17,6 +20,35 @@ def run_tests(self):
         errno = pytest.main(self.test_args)
         sys.exit(errno)
 
+
+class install_lib(_install_lib):
+    """
+        install_lib class extended to compile the locales
+        from the .po source files in .mo files.
+    """
+    def run(self):
+        _install_lib.run(self)
+        try:
+            # django before 1.7
```

Comment:
@allo- can you update to remove the django 1.7 changes since we no longer support this version?